components/text_normalization/src/main.py

```diff
@@ -44,18 +75,25 @@ def transform(self, dataframe: pd.DataFrame) -> pd.DataFrame:
         Returns:
             Pandas dataframe
         """
+        if self.do_lowercase:
+            dataframe[("text", "data")] = dataframe[("text", "data")].apply(lambda x: x.lower())
+
+        if self.remove_bad_patterns:
+            dataframe[("text", "data")] = dataframe[("text", "data")].apply(remove_noisy_lines)
```

Comment:
Shouldn't these be swapped? If you're lower casing first, you're no longer removing the bad uppercase pattern. Or is this desired behavior?